src/main/java/no/nav/pam/annonsemottak/receivers/solr/fetch/StopSolrStillingerService.java

```diff
@@ -24,6 +29,19 @@
     private final MeterRegistry meterRegistry;
     private final StillingRepository stillingRepository;
 
+    /**
+     * @deprecated Fjern når alle solr-stillinger er utløpt. Da er denne ikke akutell lenger
+     */
+    @Deprecated
+    private Predicate<Stilling> ignoreOverfortFraArbeidsgiver = s -> {
+
+        if(toggle("pam.schedule.fetch.from.xmlstilling").isDisabled()) {
+            return !s.getMedium().equals(fraArbeidsgiver);
+        }
+
+        return true; // accept all if not not fra arbeidsgiver and toggle not disabled
```

Comment:
Bytte om logikk her, så er det vel good to go :).

Reply:
Skal være bytta nå